addons/common/functions/fnc_findUnloadPosition.sqf

```diff
@@ -40,12 +40,18 @@ if (_checkVehicleIsStable) then {
 };
 
 private _radiusOfItem = 1;
-if (_typeOfCargo isKindOf "CAManBase") then {
+if (_cargo isKindOf "CAManBase") then {
     _radiusOfItem = 1.1;
 } else {
     //`sizeOf` is unreliable, and does not work with object types that don't exist on map, so estimate size based on cargo size
-    if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
-        _radiusOfItem = (((getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) ^ 0.35) max 0.75);
+    private _typeOfCargo = if (_cargo isEqualType "") then {_cargo} else {typeOf _cargo};
+    private _itemSize = if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
+        getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size));
+    } else {
+        [_cargo] call EFUNC(cargo,getSizeItem);
```

Comment:
there is error here if cargo module is no loaded

Reply:
`[[_cargo] call EFUNC(cargo,getSizeItem)] param [0, _radiusOfItem];` could resolve that issue.

Reply:
I meant if cargo module is not loaded there is no `EFUNC(cargo,getSizeItem)` and you call non-existing function

Reply:
I just learned in scheduled EFUNC(cargo,getSizeItem) will trow an undefined variables which is not the case in unscheduled.
If you guys are using an other approach, let me know, I will use it for consistency.